anime-list-master.xml

```diff
@@ -41743,7 +41743,7 @@
   <anime anidbid="15144" tvdbid="388737" defaulttvdbseason="1" episodeoffset="" tmdbid="" imdbid="">
     <name>Jaku-chara Tomozaki-kun</name>
   </anime>
-  <anime anidbid="15145" tvdbid="" defaulttvdbseason="" episodeoffset="" tmdbid="" imdbid="">
+  <anime anidbid="15145" tvdbid="371000" defaulttvdbseason="1" episodeoffset="" tmdbid="94498" imdbid="tt11714178">
```

Comment:
This tmdbid belongs to https://www.themoviedb.org/movie/94498-i-love-dollars

Reply:
Ooops, no idea why I added tmdb/imdb there. No need for those. Just tvdbid for tv show is enough. Fixed.